components/developer/pkg-config/Makefile

```diff
@@ -24,27 +24,27 @@
 # Copyright (c) 2017, Aurelien Larcher
 #
 
+BUILD_STYLE=configure
+BUILD_BITS=64
 include ../../../make-rules/shared-macros.mk
```

Comment:
This breaks firefox:
checking for ar... /usr/gnu/bin/ar
checking for pkg_config... not found
DEBUG: pkg_config: Looking for /usr/bin/amd64/pkg-config
ERROR: Cannot find pkg_config
gmake: *** [/export/home/andreas/oi-userland/make-rules/configure.mk:175: /export/home/andreas/oi-userland/components/web/firefox/build/amd64/.configured] Error 1

This seems to be coming from the latest pkg-config changes which aren't present yet.

Reply:
Well the firefox Makefile is incorrect:
`echo "ac_add_options PKG_CONFIG=/usr/bin/amd64/pkg-config" >> $(MOZCONFIG) ;`

The PATH is set so that the right pkg-config is used in 64-bit mode, hardcoding the path is wrong.

Reply:
This does not make sense as both pkg-config binaries are identical.

Reply:
I see, we need to fix it later. I have already merged the change because it works without your developer-fixes.

Reply:
No problem, this is not a big deal :)